lib/swipe_image_gallery.dart

```diff
@@ -319,6 +323,7 @@ class SwipeImageGallery<T> {
                   setBackgroundOpacity: setOpacity,
                   dragEnabled: dragEnabled,
                   children: children,
+                  useSafeArea: useSafeArea,
```

Comment:
Can you move the `children` to the bottom here to adhere with [sort_child_properties_last](https://dart.dev/diagnostics/sort_child_properties_last)? This will also fail the CI run linter check.